calico/acl_manager/test/test_processor.py

```diff
@@ -53,6 +54,22 @@ def tearDown(self):
                          'outbound': [],
                          'outbound_default': 'deny'}}
 
+    def assertDictEqual(self, d1, d2, msg=None):
+        """
+        This override for assertDictEqual ensures that lists do not need to
+        be ordered for dict comparison. It's an unfortunate hack, but c'est la
+        vie.
+        """
+        for k,v1 in d1.iteritems():
+            self.assertIn(k, d2, msg)
+            v2 = d2[k]
+            if (isinstance(v1, collections.Iterable) and
+                not isinstance(v1, basestring)):
+                self.assertItemsEqual(v1, v2, msg)
+            else:
+                self.assertEqual(v1, v2, msg)
+        return True
```

Comment:
assertDictEqual shouldn't really return anything - it's probably harmless enough, but may as well remove so it matches the standard assertDictEqual.